unit_test/sparse/Test_Sparse_spmv.hpp

```diff
@@ -707,7 +707,6 @@ TEST_F( TestCategory,sparse ## _ ## spmv_mv_struct ## _ ## SCALAR ## _ ## ORDINA
  && defined (KOKKOSKERNELS_INST_OFFSET_SIZE_T) ) || (!defined(KOKKOSKERNELS_ETI_ONLY) && !defined(KOKKOSKERNELS_IMPL_CHECK_ETI_CALLS))
  EXECUTE_TEST(float, int, size_t, TestExecSpace)
```

Comment:
I think you already told me and I forgot, but what was the rationale for removing the test?

Reply:
The macro for that test was undefined, I think it was an artifact of a WIP PR for spmv_struct that didn't get removed and wasn't caught due to lack of float testing.